include/qinterface.hpp

```diff
@@ -1166,6 +1185,13 @@ class QInterface {
      */
     virtual void SetBit(bitLenInt qubitIndex1, bool value);
 
+    /**
+     * Act as though a measurement was applied, except force the result of the measurement
```

Comment:
This doesn’t clear up what this method does that’s different than `M()`, as it seems like that would do the same thing.

Reply:
I don't think you're reading closely enough, here, but I'll make it a little more obvious in the comment.

A quantum measurement can't be "forced." If the state is a superposition of the measurement basis states, a true measurement is probabilistic and not entirely under the control of the "experimenter" making the measurement. The difference is exactly the part about the measurement being "forced."